src/Chapter1.hs

```diff
@@ -554,7 +556,12 @@ value after "=" where the condition is true.
 Casual reminder about adding top-level type signatures for all functions :)
 -}
 
-mid x y z = error "mid: not implemented!"
+mid :: Int -> Int -> Int -> Int
+mid x y z
+    | x <= y && y <= z || z <= y && y <= x = y
+    | y <= x && x <= z || z <= x && x <= y = x
+    | x <= z && z <= y || y <= z && z <= x = z
+    | otherwise = x
```

Comment:
It looks like the third case is redundant and is already covered by `otherwise = z` 🙂

Reply:
vscode was giving exhaustivity warning, so I added regardless

Reply:
You indeed need to have `otherwise` in the end. What I was trying to say is that you can remove the `y <= x && x <= z || z <= x && x <= y = x` line entirely